resources/swiftlint.yml

```diff
@@ -22,19 +34,52 @@ whitelist_rules:
   - vertical_whitespace
   - void_return
   - custom_rules
+  - force_try
+  - legacy_constructor
+  - mark
+
+included: # paths to include during linting. `--path` is ignored if present.
+  - Source
 
-excluded:
-  - Carthage
+excluded: # paths to ignore during linting. Takes precedence over `included`.
   - Pods
 
+analyzer_rules:
+  - unused_import
+
 colon:
   apply_to_dictionaries: false
 
-indentation: 2
+indentation: 4
+
+# configurable rules can be customized from this configuration file
+# binary rules can set their severity level
+force_try:
+  severity: warning # explicitly
+# rules that have both warning and error levels, can set just the warning level
+# implicitly
+line_length:
+  warning: 300
```

Comment:
We should perhaps limit this to 150, or even 120. A line with 150+ characters is already kinda un-readable.